config/settings.d/tftp.yml.example

```diff
@@ -17,3 +17,7 @@
 # Defines the default dns timeout in seconds needed to download tftp artifacts
 # like initrd and vmlinuz. Default value 10 seconds
 #:tftp_dns_timeout: 10
+
+# Defines the default certificate action for certificate checking.
+# When false, the argument --no-check-certificate will be used.
+#:verify_server_cert: false
```

Comment:
We are aiming for a safe default, to prevent breaking existing installations. But now that I read this, it might feels weird long-term. @ekohl do you prefer this to be true or false?

Maybe we can set it to true by default and for upgrades, we can provide a false value via the installer?

Reply:
Let's make this `true` then.